scripts/ConvertToAckermann.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+"""
+Subscribes to a twist message and converts it to an ackermann message.
+Subscribes to /twist_vel
+Publishes to /cmd_vel
+
+Assumes all inputs are normalized between -1 and 1
+
+@edited: 9/28/18
```

Comment:
Make that up-to-date. Personally, the year should be all 4 letters, though there is an argument for git taking care of date annotations for you.